core/state_processor.go

```diff
@@ -120,6 +120,7 @@ func ApplyTransactionWithEVM(msg *Message, config *params.ChainConfig, gp *GasPo
 	}
 	// Create a new context to be used in the EVM environment.
 	txContext := NewEVMTxContext(msg)
+	txContext.AccessEvents = statedb.NewAccessEvents()
```

Comment:
Shouldn't this only be set if verkle is active?

Reply:
it's not a mistake, one needs to start with an empty access event list when starting a conversion.